widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadLayout.kt

```diff
@@ -39,6 +39,7 @@ internal class ChatHeadLayout @JvmOverloads constructor(
 
     private val chatHeadSize: Float by lazy { resources.getDimension(R.dimen.glia_chat_head_size) }
     private val chatHeadMargin: Float by lazy { resources.getDimension(R.dimen.glia_chat_head_content_padding) }
+    private val dragMargin by lazy { resources.getDimension(R.dimen.glia_small) }
```

Comment:
As far as I understood, it's a minimum value for the `coerceIn()` function. What do you think about naming it `minDraggingValue`, `minimumCoordinate` or whatnot? I'm suggesting it because `dragMargin` is associated with padding, space between views to me.

Reply:
This is the bubble margins from the screen borders.
I'll think how to rename it to sound more specific.

Reply:
fixed!